macros/parserGraphTool.pl

```diff
@@ -228,6 +235,17 @@ sub new {
 }
 
 our %graphObjectTikz = (
+	point => {
+		code => sub {
+			my $self = shift;
+			my ($x, $y) = @{$_->{data}[2]{data}};
+			my $point = "($x,$y)";
+			return ("\\draw[thick,blue,fill=blue!30] $point circle[radius=5pt];", [
+						$point,
+						sub { return ($_[0] - $x)**2 + ($_[1] - $y)**2; }
+					]);
```

Comment:
This will need to be changed to
```suggestion
					"($self->{bBox}[0],$self->{bBox}[1]) -- ($self->{bBox}[2],$self->{bBox}[1]) -- " .
					"($self->{bBox}[2],$self->{bBox}[3]) -- ($self->{bBox}[0],$self->{bBox}[3]) -- cycle",
					sub { return 1; }
				]);
```
Basically this just clips to the entire graph.  The previous code was clipping to the point, which prevented fills from being drawn at all.